developer-docs-site/docs/concepts/aptos-token.md

```diff
@@ -0,0 +1,172 @@
+
+## Token vs coin standards
+
+The coin standard is separated out into its own Move module to ensure that:
+
+- The coin standard can be used to create a token with decimal and with minimal metadata. 
+- The coin module remains a part of the Aptos core framework and be used for the gas currency, thereby enhancing the core functionality of the Aptos framework.
+
+The token standard, on the other hand, is deployed as a separate package at the Aptos blockchain address `0x3`. The token standard is designed to create an NFT or a semi-fungible or a fungible non-decimal token, with rich metadata and functionalities. A token definition of this type can be iterated rapidly to respond to the platform and user requirements. 
+
+:::tip Aptos token standard
+This document presents the proposed Aptos token standard. The Aptos implementation for core NFTs or tokens can be found in [token.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token/sources/token.move). Also see the tutorial [Your First NFT](/tutorials/your-first-nft.md).
+
+The coin standard is presented in a different document.
+:::
+
+### Overview of NFT
+
+An [NFT](https://en.wikipedia.org/wiki/Non-fungible_token) is a non-fungible token or data stored on a blockchain that uniquely defines ownership of an asset. NFTs were first defined in [EIP-721](https://eips.ethereum.org/EIPS/eip-721) and later expanded upon in [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155). NFTs are typically defined using the following properties:
+
+- `name`: The name of the asset. It must be unique within a collection.
+- `description`: The description of the asset.
+- `uri`: A URL pointer to off-chain for more information about the asset. The asset could be media such as an image or video or more metadata.
+- `supply`: The total number of units of this NFT. Many NFTs have only a single supply while those that have more than one are referred to as editions.
+
+Additionally, most NFTs are part of a collection or a set of NFTs with a common attribute, for example, a theme, creator, or minimally contract. Each collection has a similar set of attributes:
+
+- `name`: The name of the collection. The name must be unique within the creator's account.
+- `description`: The description of the collection.
+-  `uri`: A URL pointer to off-chain for more information about the asset. The asset could be media such as an image or video or more metadata.
+
+## Token standard
+
+The Aptos token standard is developed following the below principles:
+
+- Provide a standard implementation to improve interoperability across the ecosystem projects.
+- Achieve maximal liquidity by defining the NFT, fungible (non-decimal) and semi-fungible tokens in one contract. These different types of tokens can be easily stored, transferred and transacted in the same way.
```

Comment:
maybe this is intended to achieve maximal interoperability across marketplaces, exchanges, and other methods for exchange